javaparser-core/src/main/java/com/github/javaparser/ast/visitor/CloneVisitor.java

```diff
@@ -429,6 +429,15 @@ public Node visit(WildcardType _n, Object _arg) {
 		return r;
 	}
 
+    @Override
+    public Node visit(UnknownType _n, Object _arg) {
+        Comment comment = cloneNodes(_n.getComment(), _arg);
```

Comment:
I think not. This is a kind of null object and can't receive any comment as it has no lexical representation.